microk8s-resources/actions/common/utils.sh

```diff
@@ -219,6 +219,46 @@ skip_opt_in_config() {
     fi
 }
 
+# Function to sanitize arguments for API server
+sanatize_args_kubeapi_server() {
+  local deprecated_args=("insecure-port" "feature-gates=RemoveSelfLink")
```

Comment:
Please try to handle all the arguments that got removed from lets say 1.20 to 1.26 here. This way we would cleanly handle arguments inherited from old k8s releases.